Detectors/HMPID/calibration/src/HMPIDDCSProcessor.cxx

```diff
@@ -483,16 +482,18 @@ bool HMPIDDCSProcessor::evalCorrFactor(double dRefArgon, double dCellArgon,
 
   // evaluate 15 mm of thickness C6F14 Trans
 
-  aConvFactor = 1.0 - 0.3 / 1.8;
+  // ef: check if all are not nullpointers
+  if (&dRefArgon == nullptr || &dCellArgon == nullptr || &dRefFreon == nullptr || &dRefFreon == nullptr) {
```

Comment:
The macOS CI complaints are genuine: the address of the reverence cannot be nullptr. You should probably check a specific value (passing a pointer seems to be meaningless since the method is called with the reference of double.